frameworks/g-cloud-9/questions/services/incidentManagementType.yml

```diff
@@ -0,0 +1,22 @@
+name: "Incident management type"
+question: "Which incident management processes does your organisation comply with?"
+depends:
+  -
+    "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: radios
+options:
+    -
+        label: "Conforms to a recognised standard, for example, CSA CCM v3.0 or ISO/IEC 27035:2011 or SSAE-16 / ISAE 3402"
+    -
+        label: "Supplier-defined controls"
+    -
+        label: "Undisclosed"
```

Comment:
Undisclosed is used here to suggest that they implement some sort of process, but for whatever reason aren't willing to disclose what it is.